test/models/vote_test.rb

```diff
@@ -0,0 +1,177 @@
+require "test_helper"
+
+describe Vote do
+  let (:new_vote) {
+    Vote.create(work_id: works(:new_work).id, user_id: users(:new_user).id, vote_type: "upvote")
+  }
+  
+  let (:new_work2) {
+    Work.create(category: "book", title: "Cool Book2", creator: "Cooler Author", description: "Here is a desc", publication_year: 1993, vote_count: 0)
+  }
+
+  let (:new_work3) {
+    Work.create(category: "book", title: "Cool Book3", creator: "Cooler Author", description: "Here is a desc", publication_year: 1993, vote_count: 0)
+  }
+
+  let (:new_work4) {
+    Work.create(category: "book", title: "Cool Book4", creator: "Cooler Author", description: "Here is a desc", publication_year: 1993, vote_count: 0)
+  }
+
+  it "can be instantiated" do
+    assert(new_vote.valid?)
+  end
+
+  it "will have the required fields" do
+    new_vote.save
+    [:work_id, :user_id].each do |field|
+      expect(new_vote).must_respond_to field
+    end
+  
+  end
+
+  describe "relationships" do
+    
+    it "can have one work" do
+      expect(new_vote.work).must_be_instance_of Work
+      expect(new_vote.work.title).must_equal "Cool Book"
+    end
+
+    it "can have one user" do
+      expect(new_vote.user).must_be_instance_of User
+      expect(new_vote.user.username).must_equal "Emily"
+    end
+  
+  end
+
+  describe "validations" do
+    
+    it "must have a work_id" do
+      new_vote.work_id = nil
+
+      refute(new_vote.valid?)
+      expect(new_vote.errors.messages).must_include :work_id
+      expect(new_vote.errors.messages[:work_id]).must_equal ["can't be blank"]
+    end
+    
+    it "must have a user_id" do
+      new_vote.user_id = nil
+
+      refute(new_vote.valid?)
+      expect(new_vote.errors.messages).must_include :user_id
+      expect(new_vote.errors.messages[:user_id]).must_equal ["can't be blank"]
+    end
+  
+    it "must have a vote_type" do
+      new_vote.vote_type = nil
+
+      refute(new_vote.valid?)
+      expect(new_vote.errors.messages).must_include :vote_type
+      expect(new_vote.errors.messages[:vote_type]).must_equal ["can't be blank"]
+    end
+
+    it "must have a unique user_id in the score of the work_id" do
+      new_vote.save
+      new_vote2 = Vote.create(work_id: works(:new_work).id, user_id: users(:new_user).id, vote_type: "upvote")
+      refute(new_vote2.valid?)
```

Comment:
You've got a lot of assert-style expectations.

```suggestion
     expect(new_vote2.valid?).wont_equal true
```